server/src/main/resources/application.properties

```diff
@@ -130,3 +130,28 @@ management.datadog.metrics.export.enabled=false
 # management.datadog.metrics.export.apiKey=YOUR_API_KEY
 # management.datadog.metrics.export.uri=dd instance URL
 # management.datadog.metrics.export.step=10s
+
+# To enable webhook module for TaskStatus and WorkflowStatus notifications
+conductor.workflow-status-listener.type=workflow_publisher
+conductor.task-status-listener.type=task_publisher
+
+# To enable Workflow/Task Summary Input/Output JSON Serialization, use the following:
+conductor.app.summary-input-output-json-serialization.enabled=true
+
+# Webhook Push notification properties
+conductor.status-notifier.notification.url=
+conductor.status-notifier.notification.endpointWorkflow=
+conductor.status-notifier.notification.endpointTask=
+# Use enums in TaskModel.Status
```

Comment:
these properties should be commented out by default so users can enable if required.